addons/sale/tests/test_onchange.py

```diff
@@ -65,6 +65,51 @@ def test_onchange_product_id(self):
         # Check the unit price of SO line
         self.assertEquals(100, sale_order.order_line[0].price_unit, "The included tax must be subtracted to the price")
 
+    def test_fiscalposition_application(self):
+        """Test application of a fiscal position mapping
+        price included to price included tax
+        """
+
+        uom_id = self.product_uom_model.search([('name', '=', 'Units')])[0]
+        pricelist = self.pricelist_model.search([('name', '=', 'Public Pricelist')])[0]
+
+        partner_id = self.res_partner_model.create(dict(name="George"))
+        tax_include_src_id = self.tax_model.create(dict(name="Include tax",
+                                                    amount='21.00',
+                                                    price_include=True,
+                                                    type_tax_use='sale'))
+        tax_include_dst_id = self.tax_model.create(dict(name="Exclude tax",
+                                                    amount='6.00',
+                                                    price_include=True,
+                                                    type_tax_use='sale'))
```

Comment:
That's a personnal point of view but this looks disgusting to me. What about
```py
tax_include_dst = self.env['account.tax'].create({
    'name': "Exclude tax",
    'amount': 6.0,
    'price_include': True,
    'type_tax_use': 'sale',
}
```
instead? Note:
- a variable should end with `_id` only if it's an id... not a record.
- `amount` is a float field, not a char one
- `type_tax_use` is already `sale` by default.
- You are loosing a looooot of empty space on the left

Reply:
I agree, my first attempt was to follow the file style :)